FFMpegCore.Test/FFMpegArgumentProcessorTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FluentAssertions;
+
+namespace FFMpegCore.Test
+{
+    [TestClass]
+    public class FFMpegArgumentProcessorTest
+    {
+
+        private static FFMpegArgumentProcessor CreateArgumentProcessor() => FFMpegArguments
+                        .FromFileInput("")
+                        .OutputToFile("");
+
+
+        [TestMethod]
+        public void Processor_GlobalOptions_GetUsed()
+        {
+
+            var globalWorkingDir = "Whatever";
+            GlobalFFOptions.Configure(new FFOptions { WorkingDirectory = globalWorkingDir });
```

Comment:
Running this line will break other tests that are getting executed after this one